### ISSUE TITLE
Heroku Error -> "Free app running time quota exhausted" 🤦

### ISSUE THREAD

**Issue by nelsonic** [dwyl/email#38]
https://dwylmail.herokuapp.com
![image](https://user-images.githubusercontent.com/194400/88642283-316cdf80-d0b8-11ea-871d-63264657570b.png)

```
heroku logs --tail
```
![image](https://user-images.githubusercontent.com/194400/88642421-5feaba80-d0b8-11ea-9ce7-b74e0d9b6739.png)

$7/month ... 💭 
![image](https://user-images.githubusercontent.com/194400/88642720-bf48ca80-d0b8-11ea-9de0-917c09639e0b.png)

**Comment by nelsonic** [comment]
A potential (_short-term_) solution could be to scale up the dyno before test and scale it back down once tests pass.

Run the command:
```
heroku ps:type hobby
```
Output:
```
Scaling dynos on ⬢ dwylmail... done
type  size   qty  cost/mo
────  ─────  ───  ───────
web   Hobby  1    7
```

Confirmed via Web UI:
<img width="1361" alt="heroku-dyno-hobby-7-month" src="https://user-images.githubusercontent.com/194400/88647980-7562e300-d0be-11ea-914a-00c674c0b8cb.png">

To scale back down:
```
heroku ps:type free
```
Output:
```
Scaling dynos on ⬢ dwylmail... done
type  size  qty  cost/mo
────  ────  ───  ───────
web   Free  1
```

Obviously it's lame to do this _manually_ each time we run a script. 💭 

Given that Email is core to our infrastructure, it might be time to just migrate it to AWS EC2: https://github.com/dwyl/aws-ses-lambda/issues/21

**Comment by tsangaris** [comment]
Had the same issue and migrated all my sites to Vercel. Hopefully, there is no such cap there.

**Comment by nelsonic** [comment]
Yeah, Vercel is a good option for JS apps and they have a more liberal fair-use policy. 👍 
This App is built with Elixir so I don't think Vercel is an option for now. 😞

**Comment by th0mas** [comment]
fly.io could also be a good option and have a generous free tier. They have an option to import apps from Heroku as well: https://fly.io/heroku
